appium-cookies/test/cookie-specs.js

```diff
@@ -0,0 +1,78 @@
+// transpile:mocha
+
+import Cookie from '../lib/cookie.js';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import 'mochawait';
+import 'should';
+
+chai.should();
+chai.use(chaiAsPromised);
+chai.expect();
+let expect = chai.expect;
+
+describe('cookies.js', () => { 
+  it("should properly read a cookie and return the cookie's value given the key", () => {
+    let cookieMan = new Cookie('k=v');
+    cookieMan.readCookie('k').should.equal('v');
+  });
+
+
+  it ('should return undefined for an undefined key value', () => {
+    let cookieMan = new Cookie('k=v');
+    expect(cookieMan.readCookie('someKey')).to.be.undefined;
+  });
+
+
+  it ('should properly decode an encoded key value pair',  () => {
+    let cookieMan = new Cookie(encodeURIComponent(' c') + '=' + encodeURIComponent(' v'));
+    cookieMan.readCookie (' c').should.equal(' v');
```

Comment:
`cookieMan.readCookie (' c').should.equal(' v');` extra space here. Should be `cookieMan.readCookie(' c').should.equal(' v');`